packages/quantic/force-app/main/default/lwc/quanticPager/e2e/pagerObject.ts

```diff
@@ -0,0 +1,72 @@
+import {Page} from '@playwright/test';
+import {isUaCustomEvent} from '../../../../../../playwright/utils/requests';
+
+export class PagerObject {
+  constructor(public page: Page) {
+    this.page = page;
+  }
+
+  get nextPageButton() {
```

Comment:
Should we be explicit in the types all of these return?
Personally I feel this makes the file more readable what do you think?


```suggestion
  get nextPageButton(): Locator {
```